ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm.robot

```diff
@@ -170,6 +170,7 @@ Verify Model Upgrade Using Canaray Rollout
     ...    n_queries=${0}
     Traffic Should Be Redirected Based On Canary Percentage    exp_percentage=${canary_percentage}
     ...    isvc_name=${isvc_name}    model_name=${model_name}    namespace=${test_namespace}
+    ...    runtime=caikit-tgi-runtime
```

Comment:
missing an `s`

Reply:
thanks! done. Running the test from local

Reply:
tested and passed